deploy/helm_charts/envs/dev.yaml

```diff
@@ -42,14 +42,10 @@ serviceGroups:
 nl:
   enabled: true
   vertex_ai_models:
-    dc-all-minilm-l6-v2-model:
-      project_id: datcom-website-dev
-      location: us-central1
-      prediction_endpoint_id: "8518340991868993536"
     uae-large-v1-model:
-      project_id: datcom-website-dev
+      project_id: datcom-nl
       location: us-central1
-      prediction_endpoint_id: "4910007712298827776"
+      prediction_endpoint_id: "1400502935879680000"
     sfr-embedding-mistral-model:
```

Comment:
Do you plan on adding index for sfr?

Reply:
will add later and they will be memory index instead of vertex index endpoint